### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by ho-oto**

### CODE REVIEW

**Review by mateuszbaran — APPROVED** [review]
Thanks!

File: src/manifolds/Grassmann.jl [hunk 1]
```diff
@@ -365,7 +365,7 @@ Compute the QR-based retraction [`QRRetraction`](@ref) on the
 ````
 where D is a $m × n$ matrix with
 ````math
-D = \operatorname{diag}( \operatorname{sgn}(R_{ii}+0,5)_{i=1}^n ).
+D = \operatorname{diag}( \operatorname{sgn}(R_{ii}+0.5)_{i=1}^n ).
```
Comment (sethaxen) [inline src/manifolds/Grassmann.jl:368]: I'm partial to using fractions:
```suggestion
D = \operatorname{diag}\left( \operatorname{sgn}\left(R_{ii}+\frac{1}{2}\right)_{i=1}^n \right).
```

**Review by sethaxen — APPROVED** [review]

File: src/manifolds/Grassmann.jl (hunk 1 above)
Comment (kellertuer) [reply src/manifolds/Grassmann.jl]: I would be a fan for using `\bigl` or `\Bigl` (and `\Bigr` of course) instead of `\left` and ´\right`, if that's ok.

Comment (kellertuer) [reply src/manifolds/Grassmann.jl]: And sure- if I am the only one favouring manually scaled brackets we can also leave it at left/right.

**Pull request merged**